cypress/integration/page.spec.js

```diff
@@ -396,7 +425,7 @@
       });
 
       it('should show posts loader while waiting for API response', () => {
-        page.mockUser1Posts()
+        page.mockUser1Posts();
```

Comment:
The `page.mockUser1Posts()` call should be followed by `.as('user1PostsRequest')` to ensure consistency with other mock calls.